api/api.proto

```diff
@@ -2217,6 +2217,14 @@ service OpenStorageCredentials {
         get: "/v1/credentials/validate/{credential_id}"
       };
     }
+  // RemoveReferences is used to remove references to credentials
+  rpc RemoveReferences(SdkCredentialRemoveReferencesRequest)
```

Comment:
I would structure this as DeleteReferences and add an API for GetReferences

The REST API paths could be DELETE /v1/credentials/references/{cred_id} and GET /v1/credentials/references/{cred_id}

Reply:
Done